src/components/components/ReinforcementStatement/ReinforcementStatement.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import "./ReinforcementStatement.scss";
+import ReinforcementImage from "./assets/reinforcementImage.jpg";
```

Comment:
you might also want to rename your folder Assets => assets , and name of the jpg file in camelCase